windows_amd64_upstream_dependencies.repos

```diff
@@ -19,8 +19,8 @@ repositories:
     version: galactic
   xacro:
     type: git
-    url: https://github.com/wmmc88/xacro.git # FORK. TODO: switch back to ros2 branch on upstream once https://github.com/ros/xacro/pull/304 merges and releases
```

Comment:
I see that the linked PR is merged so I assume changing back if fine but maybe this breaks something else down the line? Might be worth leaving a comment in there in case

Reply:
Leaving a comment in there